src/en/news/blog/2025/rgw-improving-bucket-notifs/index.md

```diff
@@ -148,13 +166,8 @@ aws --region=default --endpoint-url http://localhost:8000 s3api put-bucket-notif
   --bucket bk000000000000 --notification-configuration='{"TopicConfigurations": [{"Id": "notif1", "TopicArn": "arn:aws:sns:default::fishtopic0", "Events": []}]}'
 ```
 
-* Run hsbench again
-```
-hsbench -a 0555b35654ad1656d804 -s h7GhxuBLTrlhVUyxSPUKUV8r/2EI4ngqJxD7iBdBYLhwluN30JaT3Q== \
-  -u http://localhost:8000 -bp bk -m pd -t 64 -z 4K -l 1 -b 1
-```
 
-* Now with code that supports sharded queue. Run baseline setting with num_shards options as 1. 
+* Run baseline setting with `rgw_bucket_persistent_notif_num_shards` set to 1. 
```

Comment:
IMO, better to give the "basline" vstart comand first, and then the "default" command from line 150.